tests/integration/conftest.py

```diff
@@ -45,11 +45,12 @@ def pytest_runtestloop(session):
     for item in session.items:
         if item.parent and item.parent.cls:
             test_classes.add(item.parent.cls)
-        # OpenSearch is a pytest, not a unit test class, therefore we check based on the item parent's name
-        # (test_opensearch.py).
-        if "opensearch" in str(item.parent).lower():
+        # OpenSearch/Elasticsearch are pytests, not unit test classes, so we check based on the item parent's name.
+        # Any pytests that rely on opensearch/elasticsearch must be special-cased by adding them to the list below
+        parent_name = str(item.parent).lower()
+        if any(opensearch_test in parent_name for opensearch_test in ["opensearch", "firehose"]):
```

Comment:
I'm not 100% sure if this is actually necessary or if my comment is even accurate 😅

Reply:
This section is necessary to start the installation of OpenSearch / ElasticSearch as soon as the test execution starts (before it even reached the respective test). This speeds up the execution of the tests.
You are absolutely right that this should be done for all tests that depend on OpenSearch or Elasticsearch. 👍🏽